Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -782,7 +811,7 @@ def compute_sizes(self):
         return size_data
 
     def update_sizes(self):
-        self.master.prepare_data()
+        self.sparse_to_dense()
```

Comment:
Don't we also need the `self.set_data` call here? Same for a few lines below.

Reply:
Yes, after `self.set_data(self.sparse_to_dense())` .